### ISSUE TITLE
Add a note about the updated CRS options in the 1.4 specification to the appropriate section of the docs. 

### ISSUE THREAD

**Issue by grantbrown** [laspy/laspy#11]
The documentation could be clearer about this.

**Comment by WilliamWalker** [comment]
I wasn't sure where to ask a question, but this seemed appropriate.  Is there a way to read/write Global Encoding bits?  I've used a WKT VLR to store projection information in the header of LAS 1.4 files, but I am unsure of whether the WKT Global Encoding bit has been set to True or '1' (an ASPRS requirement).  I see in lines 692-698 of header.py that you can set_wkt().  Is this used to set the global encoding bit?

**Comment by grantbrown** [comment]
The global encoding data is a full two bytes, containing the following sub byte fields:
* GPS Time Type [0]
* Waveform Data Packets Internal [1]
* Waveform Data Packets External [2]
* Return numbers have been synthetically generated [3]
* WKT [4]
* Reserved [5:15]

You can modify this unsigned short integer arbitrarily using the property *file.header.global_encoding*, or you can access/modify the individual components via the properties:
* file.header.gps_time_type
* file.header.waveform_data_packets_internal
* file.header.waveform_data_packets_external
* file.header.synthetic_return_num
* file.header.wkt


TL;DR; yes, you can use the wkt property to set that particular flag.

**Comment by WilliamWalker** [comment]
Perhaps I am missing something, but when I try the following, the bit is not updated.

```
>>> f = laspy.file.File(las_file, mode='rw')
>>> f.header.wkt
'0'
>>>f.header.wkt = '1'
>>>f.header.wkt
'0'
>>>f.header.wkt = 1
>>>f.header.wkt
'0'
```

**Comment by grantbrown** [comment]
Huh, you're correct. It looks like this is a gap in our test coverage, so we may have missed a regression when the sub-byte fields were recently overhauled. I'll look into it.

**Comment by grantbrown** [comment]
Fixed in [eddb0a](https://github.com/grantbrown/laspy/commit/eddb0a2eefde54976c4779898b5276836e0a8e7d), several setters for the global encoding sub byte fields had problems. I've also added a test  to cover this case. Thanks for catching this.

**Comment by WilliamWalker** [comment]
Excellent!  Glad I could help.  Thanks again.
